skiko/src/commonMain/kotlin/org/jetbrains/skia/BackendRenderTarget.kt

```diff
@@ -16,16 +18,10 @@ class BackendRenderTarget internal constructor(ptr: NativePointer) : Managed(ptr
             fbFormat: Int
         ): BackendRenderTarget {
             Stats.onNativeCall()
-            return BackendRenderTarget(
-                _nMakeGL(
-                    width,
-                    height,
-                    sampleCnt,
-                    stencilBits,
-                    fbId,
-                    fbFormat
-                )
-            )
+            loadOpenGLLibrary()
```

Comment:
Do we ever create `BackendRenderTarget` before initialising context? From a first look this one seems redundant.

Reply:
Seems it isn't needed indeed. But not because we init context. Because we can create it without context and even without OpenGL. Checked it just now.